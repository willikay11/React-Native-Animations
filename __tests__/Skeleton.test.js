import React from 'react';
import {shallow} from 'enzyme';
import SkeletonLoader from '../skeleton';

describe('Skeleton loading', () => {
    describe('Rendering', () => {
        it('should match to snapshot - square', () => {
            const square = shallow(<SkeletonLoader type='square' loading={true} />)
            expect(square).toMatchSnapshot()
        });

        it('should match to snapshot - rectange', () => {
            const rectangle = shallow(<SkeletonLoader type='rectangle' loading={true} />)
            expect(rectangle).toMatchSnapshot()
        });

        it('should match to snapshot - circle', () => {
            const circle = shallow(<SkeletonLoader type='circle' loading={true} />)
            expect(circle).toMatchSnapshot()
        });
    });
});