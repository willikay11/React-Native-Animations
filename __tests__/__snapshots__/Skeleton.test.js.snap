// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Skeleton loading Rendering should match to snapshot - circle 1`] = `ShallowWrapper {}`;

exports[`Skeleton loading Rendering should match to snapshot - rectange 1`] = `ShallowWrapper {}`;

exports[`Skeleton loading Rendering should match to snapshot - square 1`] = `ShallowWrapper {}`;

exports[`Skeleton loading Rendering should match to snapshot 1`] = `ShallowWrapper {}`;
